lib/Notes/components/NotesAccordion/components/NotesAssigningModal/NotesAssigningModal.js

```diff
@@ -126,11 +126,15 @@ class NotesAssigningModal extends React.Component {
   };
 
   firstColumnHeaderFormatter = {
+    [columnNames.TITLE]: (note) => ((
+      <span className="note-title">{note.title}</span>
+    )),
```

Comment:
You can remove redundant parentheses to get rid of a code smell.
```suggestion
    [columnNames.TITLE]: (note) => (
      <span className="note-title">{note.title}</span>
    ),
```